syne_tune/backend/local_backend.py

```diff
@@ -243,8 +260,12 @@ def _file_path(self, trial_id: int, filename: str):
 
     def _write_time_stamp(self, trial_id: int, name: str):
         time_stamp_path = self._file_path(trial_id=trial_id, filename=name)
-        with open(time_stamp_path, 'w') as f:
-            f.write(str(datetime.now().timestamp()))
+        try:
+            with open(time_stamp_path, 'w') as f:
+                f.write(str(datetime.now().timestamp()))
+        except OSError as exc:
```

Comment:
Why is this one needed?

Reply:
I had this exception being thrown. But I can take this out again, because if the local disk fills up, things will break anyway.

Reply:
Ok can we remove it then? It seems that this should be an error then if the file cant be written.